apps/api/src/domain/models/photograph/queries/photograph-queries-fetch-by-id.e2e.spec.ts

```diff
@@ -70,8 +70,6 @@ const existingMediaItem = getValidAggregateInstanceForTest(AggregateType.mediaIt
     id: buildDummyUuid(55),
 });
 
-const testPhotographer = 'Jeff Thomas';
-
 // const photograghHeightPx = 600;
```

Comment:
Why are these commented out?

Reply:
Uncommented and reintegrated into the test